src/domains/ecommerce-app/04_apim_ecommerce.tf

```diff
@@ -438,7 +456,7 @@ module "apim_pagopa_ecommerce_helpdesk_service_api_v1" {
   name                  = "${local.project}-helpdesk-service-api"
   api_management_name   = local.pagopa_apim_name
   resource_group_name   = local.pagopa_apim_rg
-  product_ids           = [module.apim_ecommerce_product.product_id]
+  product_ids           = [module.apim_ecommerce_product.product_id,module.apim_ecommerce_helpdesk_product.product_id]
```

Comment:
Since with this PR we are adding a new ad-hoc product for helpdesk can we remove those api from eCommerce one? what do you think?